cygnus-ngsi/src/test/java/com/telefonica/iot/cygnus/sinks/NGSICartoDBSinkTest.java

```diff
@@ -48,208 +56,295 @@ public NGSICartoDBSinkTest() {
     } // NGSICartoDBSinkTest
     
     /**
-     * [NGSICartoDBSink.configure] -------- Configured 'endpoint' cannot be null.
+     * [NGSICartoDBSink.configure] -------- Independently of the configured value, enable_lowercase is always 'true'
+     * by default.
      */
     @Test
-    public void testConfigureEndpointIsNotNull() {
+    public void testConfigureEnableLowercaseAlwaysTrue() {
         System.out.println(getTestTraceHead("[NGSICartoDBSink.configure]")
-                + "-------- Configured 'endpoint' cannot be null");
-        String endpoint = null;
+                + "-------- Independently of the configured value, enable_lowercase is always 'true' by default");
+        String endpoint = "https://localhost";
         String apiKey = "1234567890abcdef";
         String dataModel = null; // default one
-        String enableLowercase = null; // default one
+        String enableLowercase = "false";
         String flipCoordinates = null; // default one
         String enableRaw = null; // default one
         String enableDistance = null; // default one
+        String keysConfFile = "/keys.conf";
         NGSICartoDBSink sink = new NGSICartoDBSink();
         sink.configure(createContext(endpoint, apiKey, dataModel, enableLowercase, flipCoordinates, enableRaw,
-                enableDistance));
+                enableDistance, keysConfFile));
         
         try {
-            assertTrue(sink.invalidConfiguration);
+            assertTrue(sink.enableLowercase);
             System.out.println(getTestTraceHead("[NGSICartoDBSink.configure]")
-                    + "-  OK  - null value detected for 'endpoint'");
+                    + "-  OK  - 'enable_lowercase=false' was confiured, nevertheless it is always true by default");
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1033/commits/948967e125859e6f768e8175076ab9ad1a815d9e